source/programming/python.rst

```diff
@@ -12,3 +12,7 @@ Python 语言
     - `X 分钟速成 Python <https://learnxinyminutes.com/docs/zh-cn/python-cn/>`__\ （简要）
     - `A Visual Intro to NumPy and Data Representation <https://jalammar.github.io/visual-numpy/>`__\
       （简要、形象地介绍 NumPy 数据结构）
+
+Linux/macOS 都内置了 Python，足够日常使用，但强烈建议不要使用系统内置的 Python，
+而建议安装 `Anaconda <https://www.anaconda.com/products/individual>`__，并使用
+``conda`` 管理 Python 环境。详情请参考 :doc:`Anaconda <software:anaconda/index>`。
```

Comment:
Maybe we should also add the note at https://seismo-learn.org/seismology101/exercises/introduction/?

Reply:
I was adding a new section "exercises/prerequisite.rst" after the introduction section, but I forgot to commit this file.

Reply:
Fixed in 1b5ca43605f9dfbd1e1b8a682bdfd93657b61c5f